test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM rust
```

Comment:
Nice, but I have been thinking about changing the arduino/d1mini code to use micropython and try to keep the whole project to use as much python as possible. Maybe we could use the python implementation here instead?